meta/main.yml

```diff
@@ -19,6 +19,8 @@ galaxy_info:
 dependencies:
   - role: stackhpc.os-shade
     os_shade_venv: "{{ os_projects_venv }}"
+    os_shade_upper_constraints: "{{ os_projects_upper_constraints | default(omit) }}"
```

Comment:
Does the omit work here? Might be better to have a default of `None` as we do in these roles: https://github.com/stackhpc/ansible-role-os-shade/blob/master/defaults/main.yml#L18

Reply:
Agreed, thanks!